irods/manager/user_manager.py

```diff
@@ -325,16 +326,8 @@ def getmembers(self, name):
         return [iRODSUser(self, row) for row in results]
 
     def addmember(
-        self, group_name, user_name, user_zone="", group_admin=False, **options
+        self, group_name, user_name, user_zone="", group_admin=None, **options
     ):
-
-        if not options.pop("suppress_deprecation_warning", False):
-            warnings.warn(
-                "Use of session.user_groups is deprecated in v1.1.7 - prefer session.groups",
-                DeprecationWarning,
-                stacklevel=2,
-            )
```

Comment:
Sorry, I'm a little confused by this...

What was the point of this deprecation message if it's just being removed? Shouldn't we be removing the whole function since it is deprecated? I feel like just removing the deprecation message is more of a de-deprecation than a removal.

OR, is the deprecation message no longer valid because one can no longer reference `session.user_groups` due to the changes in session.py?